examples/gpu-operator.yml

```diff
@@ -38,11 +38,13 @@ matrices:
         test_name: gpu-operator-e2e-upgrade
         operator_version: master
         variant: upgrade
+        prow_step:  2-postupgrade-gpu-operator-validate-deployment
```

Comment:
`s/:  2-/: 2-/s`
i.e. remove two spaces

Reply:
thanks for the English translation, I would have had to test to to understand `sed` substitution ;-)